Assets/GothicVR/Scripts/Properties/AbstractProperties.cs

```diff
@@ -4,6 +4,21 @@ namespace GVR.Properties
 {
     public abstract class AbstractProperties : MonoBehaviour
     {
-        public GameObject go => gameObject;
+        // using this to store the hero gameobject as monobehaviour has gameObject as readonly
+        // I did this because it's easier than to recreate the properties on every world transition
+        private GameObject customGameObject; 
+
+        public GameObject go
+        {
+            get
+            {
+                if (customGameObject != null)
+                    return customGameObject;
+
+                return gameObject;
+            }
+            // FIXME - Seems this is never called. Let's see if we can remove it later safely.
+            set => customGameObject = value;
```

Comment:
If I check in Rider, the setter is never called. If we don't need any longer, please reduce complexity and leverage default gameObject again.

Reply:
As you made the npcproperties to be to VRPlayer, we don't need the custom gameobject at all now